src/js/containers/search/visualizations/rank/RankVisualizationWrapperContainer.jsx

```diff
@@ -36,37 +32,14 @@ export class RankVisualizationWrapperContainer extends React.Component {
         super(props);
 
         this.state = {
-            spendingBy: 'budget_category',
-            budgetFiltersSelected: false,
-            awardFiltersSelected: false
+            spendingBy: 'awarding_agency'
         };
 
         this.changeSpendingBy = this.changeSpendingBy.bind(this);
     }
 
-    componentDidMount() {
-        this.setFilterStates();
-    }
-
-    componentDidUpdate(prevProps) {
-        if (!isEqual(this.props.reduxFilters, prevProps.reduxFilters)) {
-            this.setFilterStates();
-        }
-    }
-
-    setFilterStates() {
-        this.setState({
-            budgetFiltersSelected:
-                BudgetCategoryHelper.budgetFiltersSelected(this.props.reduxFilters),
-            awardFiltersSelected:
-                BudgetCategoryHelper.awardFiltersSelected(this.props.reduxFilters)
-        });
-    }
-
     generateVisualization() {
         switch (this.state.spendingBy) {
-            case 'budget_category':
-                return <SpendingByCategoryRankVisualizationSectionContainer {...this.state} />;
             case 'awarding_agency':
                 return <SpendingByAwardingAgencyVisualizationContainer {...this.state} />;
```

Comment:
🕵️ Do we still need to pass the state down the child containers?

Reply:
👨‍💻  Nope, child containers no longer need state because they no longer need to switch endpoints based on applied filters!